pathways/shipments.py

```diff
@@ -183,6 +184,63 @@ def generate_shipment(self):
         )
 
 
+class AQIMShipmentGenerator:
+    """Generate a shipments based on existing AQIM records"""
+
+    def __init__(self, quant_unit, stems_per_box, filename, separator=","):
```

Comment:
I've added documentation, but also realized we do not need this variable, so its has been removed. The unit to be used for quantity can be pulled from each AQIM record. Quantity is specified in many different units in AQIM, unlike in the F280 records we are using. We may have filtered F280 to only include stems quantities actually, but since we have a lot fewer AQIM inspections, we should accept stem or box.